Main_Project/src/main/java/com/italent/dao/LithiumMessageDAO.java

```diff
@@ -0,0 +1,51 @@
+package com.italent.dao;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+
+import com.italent.model.LithiumMessage;
+import com.italent.model.Users;
+
+public class LithiumMessageDAO {
+
+    public static SessionFactory getSessionFactory() {
+        Configuration configObj = new Configuration();
+        configObj.configure("hibernate.cfg.xml");
+        ServiceRegistry serviceRegObj = new StandardServiceRegistryBuilder().applySettings(configObj.getProperties())
+                .build();
+        SessionFactory factoryObj = configObj.buildSessionFactory(serviceRegObj);
+        return factoryObj;
+    }
+
+    public static String createRecord(LithiumMessage msg) {
+        try {
+            Session sessionObj = getSessionFactory().openSession();
+            Transaction transObj = sessionObj.beginTransaction();
+            sessionObj.saveOrUpdate(msg);
+            transObj.commit();
+            sessionObj.close();
+            return msg.getMsgId();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
remove e.printStackTrace(); and maintain in logs like logger.error("method name "+e,e);